playhouse/reflection.py

```diff
@@ -1,4 +1,9 @@
 from collections import namedtuple
+try:
+    from collections import OrderedDict
+except ImportError:
+    OrderedDict = dict
+
```

Comment:
Small thing, but could you move the namedtuple import below the try/except? That way they're alphabetized.